### ISSUE TITLE
Import biomass: Bayreuth 2014 (MISSING)

### ISSUE THREAD

**Issue by KW529** [felt0134/drought-net-tickets#96]
Anke stated in email on 4-24-2020: 

"Hi Kate, 
we are still checking. Actually, we must have 2014 control data, because they went into this pre-DroughtNet add-on paper (see attached).
Best, Anke"

**Comment by KW529** [comment]
I have still not heard back about the 2014 data (5-27-2020)

**Comment by KW529** [comment]
**Sent by  Schuchardt, Max <Max.Schuchardt@uni-bayreuth.de> on 09-23-2020**
Hi Kate,
after digging deep into our server and getting in contact with all our previous PhD students and post docs we came to a solution what happened to our 2014 drought net data.
Since we established DroughtNet in 2013 (which is our pre-treatment year), we skipped 2014 and collected data in 2015 as our Y1. 
The reason for skipping 2014 can’t be tracked back. We hope this is not a crucial problem.

If you have any more questions, let us know.

Thank you and best regards from Bayreuth.
Anke and Max
